### PR TITLE
Implementation of Trie (issue #464)

### PR DESCRIPTION
**Pull request by thakurutkarsh22**
Implementation of Trie without using Hashmap

### CODE REVIEW

**Comment by jainaman224** [comment]
Please refer the issue you are solving.

**Comment by thakurutkarsh22** [comment]
Issue  #464 
File name changed

File: Trie_without_HashMap/Trie.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+	public static void main(String[] args) {
+
+
+		Trie trie = new Trie() ;
+		trie.addWord("abc");
+		trie.addWord("abd");
+		trie.addWord("abl");
+		trie.addWord("axc");
+//		trie.display();
```
Comment (jainaman224) [inline Trie_without_HashMap/Trie.java]: Indentation issue.

**Comment by jainaman224** [comment]
@thakurutkarsh22 I am unable to understand the difference between Normal Trie and this Trie. It will be helpful if you can throw some light.

**Comment by thakurutkarsh22** [comment]
Not much difference only difference is the one boolean in Node class.

**Comment by jainaman224** [comment]
So what is without hash?

**Comment by jainaman224** [comment]
And no worries, you can pick another issue. We can have some knowledge sharing conversation here.

**Comment by jainaman224** [comment]
@thakurutkarsh22 What happened?

**Pull request closed without merging**